docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -63,15 +63,11 @@ The first option to configure is the name of your connection:
 
 - **Default** - Toggle to select as the default name in dashboard panels. When you go to a dashboard panel this will be the default selected data source.
 
-### HTTP section
+### Connection section
 
-- **URL** - The URL of your Prometheus server. If your Prometheus server is local, use `<http://localhost:9090>`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `<http://prometheus.example.orgname:9090>`.
+- **Prometheus server URL** - The URL of your Prometheus server. If your Prometheus server is local, use `<http://localhost:9090>`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `<http://prometheus.example.orgname:9090>`.
```

Comment:
I don't think the angle brackets should be present since this is in preformatted code. The URL wouldn't have these brackets. I think they were originally there to make this a kind of link in the HTML but that's not how it renders: https://grafana.com/docs/grafana/latest/datasources/prometheus/configure-prometheus-data-source/#http-section

```suggestion
- **Prometheus server URL** - The URL of your Prometheus server. If your Prometheus server is local, use `http://localhost:9090`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `http://prometheus.example.orgname:9090`.
```